.github/workflows/build_and_test.yml

```diff
@@ -152,7 +152,7 @@ jobs:
       run: rustup target add aarch64-linux-android armv7-linux-androideabi x86_64-linux-android i686-linux-android
 
     - name: Install NDK tool
-      run: cargo install cargo-ndk
+      run: cargo install cargo-ndk=1.0.0
```

Comment:
Haven't seen this syntax before, only used `--vers 1.0.0`. Maybe that'll work on the builders?

Reply:
I had just assumed that syntax worked without testing, alas no